content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
 | Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
 | Situation       |     2/0 |     4/0 | SIT                  |
 
+Question Type Minimums
+: For a given Question Type, the minimum number of questions must be asked in Questions Numbered 1-20. This excludes any questions appended with "A" or "B."
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
+
+#### Multiple Answer
+Must have exactly one inserted interrogative word.
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
```

Comment:
This is added by me. Thoughts on including it?

Reply:
It's a definite "yes" for me, but these are the kinds of super-minor substantive changes that we'll want to document well, mark, and discuss more / highlight when we present the new rulebook to the CQLT.

Reply:
That's a good question, do we want to denote "changes that we think are substantial or do not maintain functional parity?"

Reply:
Yes, we should keep a log. Maybe start a Markdown file in the root directory that just has a bulleted list of references? I don't want to pollute the rulebook content itself. But we'll need some easy way for a non-tech to at least be aware of what we think might need closer review. That said, is there anything here that's not func equiv?

Reply:
There isn't at all. I just stated the that the opposite of something that doesn't meet the requirement, does meet the requirement.

Reply:
Cool.

Reply:
And it's actually useful because some people think that they/them is a "singular" pronoun, because there's just one pronoun, which is...not....what "singular" means at all.